include/experimental/__p2642_bits/layout_padded.hpp

```diff
@@ -0,0 +1,775 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+#pragma once
+
+#include <cassert>
+#include "layout_padded_fwd.hpp"
+#include "../__p0009_bits/dynamic_extent.hpp"
+#include "../__p0009_bits/extents.hpp"
+#include "../__p0009_bits/mdspan.hpp"
+#include "../__p0009_bits/layout_left.hpp"
+#include "../__p0009_bits/layout_right.hpp"
+#include "../__p0009_bits/layout_stride.hpp"
+
+namespace std {
+namespace experimental {
+
+namespace detail {
+template<class _T>
+MDSPAN_INLINE_FUNCTION
+constexpr _T
+__find_aligned_offset(_T __alignment, _T __offset)
+{
+  if ( __alignment == 0 )
+    return _T(0);
+  else
+    return ( ( __offset + __alignment - 1 ) / __alignment) * __alignment;
```

Comment:
Would you consider always using curlies with if-else?
```suggestion
  if ( __alignment == 0 ) {
    return _T(0);
  } else {
    return ( ( __offset + __alignment - 1 ) / __alignment) * __alignment;
  }
```